### PR TITLE
sl-89: Add logo open graph and favico

### PR DESCRIPTION
**Pull request by jaroslawtt**
image

image

image

### CODE REVIEW

**Comment by jaroslawtt** [comment]
QA promised to provide description in the evening

**Comment by jaroslawtt** [comment]
image

File: frontend/src/pages/auth/auth.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ const Auth: React.FC = () => {
     <PageLayout>
       <div className={styles['page-content']}>
         <div className={styles['logo-wrapper']}>
-          <span className={styles['logo-dot']} />
+          <Icon iconName="logo" />
```
Comment (what1s1ove) [inline frontend/src/pages/auth/auth.tsx]: it should be an image

File: frontend/index.html [hunk 2]
```diff
@@ -2,10 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="./src/assets/img/logo.svg" />
```
Comment (what1s1ove) [inline frontend/index.html]: left it in the public

File: frontend/index.html [hunk 3]
```diff
@@ -2,10 +2,15 @@
...
     <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <meta property="og:title" content="SimpleLand" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://simpleland-development.herokuapp.com/" />
+    <meta property="og:image" content="https://simpleland-development.herokuapp.com/logo.og.png" />
+    <meta property="og:description" content="Described stuff here." />
```
Comment (what1s1ove) [inline frontend/index.html]: ask our QA to provide a good description

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/pages/auth/auth.tsx (hunk 1 above)
Comment (jaroslawtt) [reply frontend/src/pages/auth/auth.tsx]: You mean to use image element or I actually need to import our logo as .png file?

**Comment by jaroslawtt** [comment]
image

**Comment by jaroslawtt** [comment]
image

File: frontend/src/libs/components/header/header.tsx [hunk 4]
```diff
@@ -26,8 +27,8 @@ const Header: React.FC<Properties> = ({ user }: Properties) => {
   return (
     <header className={styles['header']}>
       <div className={styles['logo-wrapper']}>
-        <div className={styles['logo-icon']} />
-        <span className={styles['logo-text']}>logo</span>
+        <img alt="logo" src={logo} />
```
Comment (bohdan-degtiar) [inline frontend/src/libs/components/header/header.tsx]: Use common component Icon.

**Review by bohdan-degtiar — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/header/header.tsx (hunk 4 above)
Comment (jaroslawtt) [reply frontend/src/libs/components/header/header.tsx]: https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-simpleland/pull/103#discussion_r1148556422
Didn't Vlad mean to use img element instead or I am wrong?

Comment (andanf-e) [reply frontend/src/libs/components/header/header.tsx]: Let's leave it as image for now

**Review by bohdan-degtiar — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

File: frontend/src/pages/auth/auth.tsx [hunk 5]
```diff
@@ -59,7 +60,7 @@ const Auth: React.FC = () => {
     <PageLayout>
       <div className={styles['page-content']}>
         <div className={styles['logo-wrapper']}>
-          <span className={styles['logo-dot']} />
+          <img alt="logo" src={logo} />
```
Comment (what1s1ove) [inline frontend/src/pages/auth/auth.tsx]: you image component instead

File: frontend/src/libs/components/header/header.tsx (hunk 4 above)
Comment (what1s1ove) [inline frontend/src/libs/components/header/header.tsx]: same

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**